### PR TITLE
python client fixes

### PR DESCRIPTION
**Pull request by dtrawins**

### CODE REVIEW

**Review by mzegla — APPROVED** [review]

File: client/python/samples/utils/vehicle_utils.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def vehicle_postprocess(response, image_path, output_name, output_save_path):
     if not os.path.exists(output_save_path):
         print("Making directory for output: " + output_save_path)
         os.makedirs(output_save_path)
-    output_file = os.path.join(output_save_path, img_name + '.jpg')
```
Comment (rasapala) [inline client/python/samples/utils/vehicle_utils.py:34]: Why remove this ?

File: client/python/samples/README.md [hunk 2]
```diff
@@ -183,22 +187,21 @@ Image images/golden_retriever.jpeg has been classified as golden retriever with
...
-docker run -d --rm -v <PATH_TO_MODELS>/vehicle-detection:/models/vehicle-detection -e "http_proxy=$http_proxy" -e "https_proxy=$https_proxy" -p 8000:8000 -p 9000:9000 openvino/model_server:latest --model_name vehicle-detection --model_path /models/vehicle-detection --port 9000 --rest_port 8000
+docker run -d --rm -v ${PWD}/models/vehicle-detection:/models/vehicle-detection -p 8000:8000 -p 9000:9000 openvino/model_server:latest --model_name vehicle-detection --model_path /models/vehicle-detection --port 9000 --rest_port 8000 --layout NHWC
 ```
 
 
 ### Detect vehicle  <a name="detect-vehicle">
 
 
 #### **Make vehicle detection prediction using images in binary format:**
-#### **Required Model Server running with vehicle detection model:**
```
Comment (rasapala) [inline client/python/samples/README.md:201]: I think this information is crucial.

**Review by rasapala — CHANGES_REQUESTED** [review]
Minor comments.

File: client/python/samples/utils/vehicle_utils.py (hunk 1 above)
Comment (dtrawins) [reply client/python/samples/utils/vehicle_utils.py:34]: it was causing duplicate extension like road1.jpg.jpg

File: client/python/samples/README.md (hunk 2 above)
Comment (dtrawins) [reply client/python/samples/README.md:201]: in the previous step is instruction how to start ovms with vehicle detection model, running prediction is the next step

**Review by rasapala — APPROVED** [review]
LGTM

**Pull request merged**